src/ui/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,102 @@
+import { forwardRef } from "react";
+import styled from "styled-components";
+import shapesPink1Url from "./shapes-pink-1.svg";
+import shapesPink2Url from "./shapes-pink-2.svg";
+import shapesBlue1Url from "./shapes-blue-1.svg";
+import { H1, Subheading1, P } from "../Typography";
+
+type BackgroundType = "pink-1" | "pink-2" | "blue-1";
+
+const Wrap = styled.div<{
+  $backgroundType: BackgroundType;
+}>`
+  padding: 80px 0 80px 0;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-evenly;
+  color: var(--color-white);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  ${(props) =>
+    props.$backgroundType === "pink-1" &&
+    `
+    background-image: url(${shapesPink1Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "pink-2" &&
+    `
+    background-image: url(${shapesPink2Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "blue-1" &&
+    `
+    background-image: url(${shapesBlue1Url});
+    background-color: var(--color-blue);
+  `}
+`;
+
+const BlockHeader = styled.div`
+  flex-grow: 60;
+  flex-basis: 0;
+`;
+
+const BlockSpacer = styled.div`
+  flex-grow: 40;
+  flex-basis: 0;
+`;
+
+const Header = styled.div`
+  padding-left: 18%;
+  padding-right: 2%;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-basis: 0;
+`;
+
+const Subheading = styled(Subheading1)`
+  opacity: 0.75;
+  margin: 0;
+`;
+
+const Heading = styled(H1)`
+  margin: 0;
+`;
+
+const Paragraph = styled(P)`
+  padding: 0 12% 0px 0px;
+  line-height: 19.2px;
```

Comment:
unless there is a valid reason, normally we don't recommend have fraction of units, even if it appears in designer's file that normally are spcified unintentionally

Reply:
Those ones in the variables.css also do though, such as lineheight-text and lineheight-heading

Reply:
Probably because it's copy pasted from the demo site. It's my bad not double check those pasted code carefully :) Let's fix both by removing the fractions

Reply:
So lineheight-text and lineheight-heading currently are 1.4 and 1.2, what should I change them to though? Or is that okay because it's not px rather?

Reply:
Oh I see, sorry for not making to clear in my last comment, line-height without unit suffix is valid that indicating `1.2` times more than the original line-height. Another valid usecase for fractions is for example `1.234rem`, which is also indicating a ratio to original measurement.

However a fraction in absolute units like `px` would not be a common practice.

Reply:
Yes I suspected that then. Anyway I've updated the 19.2px to 19 now